src/index.mjs

```diff
@@ -190,13 +211,65 @@ async function processCondaEnvironment(filePath) {
     }
 }
 
+// Fetch modified lines in the PR
+async function getModifiedLinesFromCommitDiff(filePath) {
+    try {
+        const { stdout, stderr } = await execPromise(`git diff HEAD^ HEAD -- ${filePath}`);
```

Comment:
if you add the `--unified=0` flag this will not output any context lines

Reply:
also why are you doing `diff HEAD^ HEAD`? This is supposed to be for pull request won't that only diff the last commit?

Reply:
you might need to diff with `{context.payload.pull_request.base.ref}`